scout.md

```diff
@@ -717,9 +717,9 @@ If you would like to define a custom Scout search builder method, you may use th
     public function boot(): void
     {
         Builder::macro('count', function () {
-            return $this->engine()->getTotalCount(
-                $this->engine()->search($this)
-            );
+            $this->limit = 10;
+
+            return $this->engine()->getTotalCount($this->raw());
```

Comment:
This doesn't seem correct? The previous statement of ->search should remain.

I also think it should be noted that this only needs to be done for Meilisearch.

Reply:
Method `raw` already does the search as it was before.

See:

```
/**
 * Get the raw results of the search.
 *
 * @return mixed
 */
public function raw()
{
    return $this->engine()->search($this);
}
```

Reply:
Gotcha 👍

Reply:
I transformed this:

```
return $this->engine()->getTotalCount(
    $this->engine()->search($this);
);
```

On this:

`return $this->engine()->getTotalCount($this->raw());`

Which is exactly the same thing. I just added the limit before.

Reply:
@joaopalopes24 I still think we should revert it for now so this PR can focus on the thing it's trying to explain.